sdk/core/azure-core/src/private/package_version.hpp

```diff
@@ -20,7 +20,7 @@ namespace Azure { namespace Core { namespace _detail {
   /**
    * @brief Provides version information.
    */
-  class PackageVersion {
+  class PackageVersion final {
```

Comment:
Can you share what regex you used to find where to add the `final` keyword?

Reply:
All done by hand, searching for "`class `" and "`struct `" (with space after).
First I put final everywhere. Then I started removing it until it compiles, while reviewing whether a base class destructor, if public, is virtual.